src/pkg/transform/image.go

```diff
@@ -32,7 +32,7 @@ func ImageTransformHost(targetHost, srcReference string) (string, error) {
 	// Generate a crc32 hash of the image host + name
 	checksum := utils.GetCRCHash(image.Name)
 
-	return fmt.Sprintf("%s/%s-%d%s", targetHost, image.Path, checksum, image.TagOrDigest), nil
+	return fmt.Sprintf("%s/%s%s-zarf-%d", targetHost, image.Path, image.TagOrDigest, checksum), nil
```

Comment:
Is this a breaking change? 

Is there something we can do to make this backwards compatible / lay out instructions as to how to upgrade?

Reply:
Yes, this is breaking - the upgrade path is to run `init` again and then redeploy zarf packages on top to reupload the images.

We might be able to pull image names/tags from their registry and check to see if they have the `-zarf-` tag and retag them if we wanted.

Reply:
I added the breaking change warning that we discussed.  I may wait on the warning rollup since that would need a bit more thought/design to make make sense (after playing with it simply reprinting them feels / looks weird)